### PR TITLE
Story Package fields

### PR DESCRIPTION
**Pull request by adgad**
Add `descriptionHTML`, `contains` and `containedIn` fields to the schema so we can work with story package contents.

Change the capi type resolution to use the type returned by ES, and amend search to match.

### CODE REVIEW

**Comment by ironsidevsquincy** [comment]
How does `contains` differ to `storyPackage`?

**Comment by adgad** [comment]
@ironsidevsquincy I think I answered this in person, but for posterity:

The idea is that `contains` is a fundamental part of the package/content, whereas `storyPackage` (unfortunately named) are curated tangentially related articles.

So an example would be Robot Week - the Robot Week package would contain the 5 or so robot week articles, but might have a story package with other technology articles (although I doubt that would happen).

I think the V2 version of storyPackages may be being renamed to something more descriptive like `curatedRelatedContent` - but that's for another day.

File: server/graphql/types/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import collection from './collection';
 import concept from './concept';
 import tag from './tag';
-import { content, article, liveBlog, video } from './content';
+import { content, article, liveBlog, video, pkg } from './content';
```
Comment (i-like-robots) [inline server/graphql/types/index.js]: why not 'package'?

File: server/lib/backends/capi/utils/resolve-content-type.js [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
 };
 
 const resolveContentType = content => {
 	if (new RegExp(webUrlRegexes.liveblog).test(content.webUrl)) {
 		return 'liveblog';
-	} else if (new RegExp(webUrlRegexes.video).test(content.webUrl)) {
-		return 'video';
 	} else {
-		return 'article';
+		return content.type || 'article';
```
Comment (i-like-robots) [inline server/lib/backends/capi/utils/resolve-content-type.js]: Looks like only some very broken content is missing a `type` field http://next-elastic.ft.com/content/item/_search?q=_missing_:type

File: server/graphql/types/index.js (hunk 1 above)
Comment (adgad) [inline server/graphql/types/index.js]: It's a reserved word in JS 😞

File: server/lib/backends/capi/utils/resolve-content-type.js (hunk 2 above)
Comment (adgad) [inline server/lib/backends/capi/utils/resolve-content-type.js]: Hmm, they look like Placeholders? Or perhaps placeholders ingested before bugs were fixed. Are you suggesting to just `return content.type` ?

File: server/graphql/types/index.js (hunk 1 above)
Comment (i-like-robots) [inline server/graphql/types/index.js]: Ugh, I wonder why...

File: server/lib/backends/capi/utils/resolve-content-type.js (hunk 2 above)
Comment (i-like-robots) [inline server/lib/backends/capi/utils/resolve-content-type.js]: I think that will be fine, yeah.

I've reingested the placeholders so they now have a `type` (they're missing a `_lastUpdatedDateTime` field so they're not subject to the continuous updater). That leaves us with one broken podcast!

**Comment by i-like-robots** [comment]
FYI I've cleaned up all the content without a `type`

**Comment by adgad** [comment]
The last build is green, github just being weird. 🚢

**Pull request merged**